### DISCUSSION TITLE
How to bind sftpgo to ONE specific IP-address (one IPV4, one IPV6)

### DISCUSSION THREAD

**Discussion by LouisAtGH** [drakkan/sftpgo#1249] [Q&A]
I installed sftpgo on ubuntu and have two problems for which I hope to get some help :)

My server has multple IP-addresses, in order to serve multiple purposes. One (one IPV4 and one IPV6) are intended for sftpgo. My problem:

1) sftpgo is listening to localhost / 127.0.0.1 (8080). For me that is absolutely not OK. I need to find a way to bind sftpgo to a specific address. I did try to force that by editing the sftpgo.json, more specific by changing "address": "" to  "address": "myipv4-address", but that does not work. At least not for the gui.
Question: How to force the GUI to listen exclusively to a specific address (IPV4 and/or IPV6)

2) I did a first minimal sftpgo setup and that seems to be OK. Than I did change the sftpgo.json. Since access to that json file access is very restricted, I had to do that using ^sudo nano <sftpgo.json>^. I do not know it that did permission changes. What ever after warts my sftpgo login and definitions .... where gone. So I have to redefine then ..... which I will not do before I have solved problem 1)
Note that I would have preferred to edit the json via the gui (which is not possible as far as I know)

**Comment by drakkan** [comment]
Hi, try reading the docs (editing sftpgo.json is not recommended). I no longer provide step-by-step support, sorry. Please check the [support policy](https://github.com/drakkan/sftpgo#support-policy)

EDIT: sftpgo does not listen on localhost by default since several versions (it listens on all the available IPs), are you using an obsolete/unsupported version?

**Reply by LouisAtGH** [reply]
YEP it does read to all available IP's ....... that is exactly the problem !!!!!!!!!!!!!!!!!!!!!!

**Reply by LouisAtGH** [reply]
Sorry I misunderstood your remark ^it listens on all the available IPs^. I could change it by putting the address in the address fields. In fact I already did that. The problem of both problems was ..... I did forget / did remove an ',' in the xml. 

Adding the ',' solved it. Sorry for the mail
